build/Dockerfile

```diff
@@ -29,12 +29,14 @@ LABEL name="Mattermost Cloud" \
 
 ENV CLOUD=/mattermost-cloud/cloud \
   USER_UID=10001 \
-  USER_NAME=cloud
+  USER_NAME=cloud \
+  HOME=/mattermost-cloud
 
 RUN  apk update && apk add libc6-compat \
                            ca-certificates \
                            mandoc \
                            aws-cli \
+                           shadow \
```

Comment:
What is this for?

Reply:
To add useradd command https://pkgs.alpinelinux.org/package/edge/community/armv7/shadow